providers/gce/gce.go

```diff
@@ -185,6 +186,7 @@ type ConfigGlobal struct {
 	// aliases. The secondary range must be present on the subnetwork the
 	// cluster is attached to.
 	SecondaryRangeName string   `gcfg:"secondary-range-name"`
+	StackType          string   `gcfg:"stack-type"`
```

Comment:
Why does this need configuration? Why don't we always check both?

Reply:
This is because even if the subnet is a dual stack subnet it is possible to create a single stack GKE cluster. Today, we can configure nodes as single stack IPv4 inside the subnet and they will not get 2 IPs. However, in the future, we are going to support a single stack IPv6 cluster. When this happens, the nodes will get dual stack IP addresses but we only want to return the IPv6 address. This field will be required at that time.